content/en/tracing/trace_collection/compatibility/php.md

```diff
@@ -164,7 +164,7 @@ Instrumenting [generators][6] is not supported on PHP 5 and PHP 7.
 
 ### PCNTL
 
-Datadog does not offer support for tracing processes forked using [pcntl][7]. When a call to `pcntl_fork` is detected, tracing is disabled in the forked process. The main process can still be traced.
+Datadog does support tracing forked processes using [pcntl][7]. When a call to `pcntl_fork` is detected, a dedicated span is created and the forked process will be instrumented. This can be disabled by using `DD_TRACE_FORKED_PROCESS`. Refer to the [library configuration page][9] for more details.
```

Comment:
```suggestion
Datadog supports tracing forked processes using [pcntl][7]. When a call to `pcntl_fork` is detected, a dedicated span is created, and the forked process is instrumented. This can be disabled with `DD_TRACE_FORKED_PROCESS`. Refer to the [library configuration page][9] for more details.
```